service/controller/drainer.go

```diff
@@ -66,7 +68,7 @@ func NewDrainer(config DrainerConfig) (*Drainer, error) {
 			K8sClient:    config.K8sClient,
 			Logger:       config.Logger,
 			Resources:    resources,
-			ResyncPeriod: 2 * time.Minute,
+			ResyncPeriod: 1 * time.Minute,
```

Comment:
this puts more pressure on the api server, especially for large clusters. Was this change made on purpose?

Reply:
we have "only" 8 minutes to drain nodes otherwise the next ASG start the node replacement, so that's why